Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -209,6 +209,9 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
         promptNoButton.addTarget(self, action: "SELdidClickOptInNo", forControlEvents: UIControlEvents.TouchUpInside)
         prompt.addSubview(promptNoButton)
 
+        // otherwise the label (i.e. question) is visited by VoiceOver *after* yes and no buttons
+        prompt.accessibilityElements = [promptImage, promptLabel, promptYesButton, promptNoButton]
```

Comment:
I thought about it. I think we should not decide here whether something is an accessibility element. That should be decided by setting `isAccessibilityElement` somewhere else. So as a matter of principle, include everything that *could* be an accessibility element, and leave the ultimate decision to another part of code.